.github/dockerfiles/yscope-presto-builder.dockerfile

```
@@ -0,0 +1,143 @@
+    && apt-get clean && rm -rf /var/lib/apt/lists/* \
+    && curl --fail --location --silent --show-error --output cmake-3.28.3-linux-x86_64.tar.gz \
+        https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.tar.gz \
+    && tar -xzf cmake-3.28.3-linux-x86_64.tar.gz -C /opt \
+    && rm cmake-3.28.3-linux-x86_64.tar.gz \
+    && ln -sf /opt/cmake-3.28.3-linux-x86_64/bin/cmake /usr/local/bin/cmake \
+    && ln -sf /opt/cmake-3.28.3-linux-x86_64/bin/ctest /usr/local/bin/ctest \
+    && ln -sf /opt/cmake-3.28.3-linux-x86_64/bin/cpack /usr/local/bin/cpack
+
+# Run setup scripts - same pattern as upstream ubuntu-22.04-dependency.dockerfile
+# rpm is needed for MinIO installation (S3-compatible storage for tests)
+RUN mkdir -p /build && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)